### PR TITLE
Add Amazon Dynamodb plug-in (GDP only)

### PR DESCRIPTION
**Pull request by Tal-Daniel**
Adds Amazon DynamoDB plugin (+offline package ZIP); 
Mainly developed by Rasika Shete, IBM.

### CODE REVIEW

**Review by IBM-klahn — APPROVED** [review]
Looks good.  Thanks for making updates to the Available plugins table.

File: filter-plugin/logstash-filter-dynamodb-guardium/build.gradle [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+// ===========================================================================
+// plugin info
+// ===========================================================================
+group                      'com.ibm.guardium.dynamodb' // must match the package of the main plugin class
+version                    "${file("VERSION").text.trim()}" // read from required VERSION file
+description                = "DynamoDB-Guardium filter plugin"
+pluginInfo.licenses        = ['Apache-2.0'] // list of SPDX license IDs
+pluginInfo.longDescription = "This gem is a Logstash Dynamodb filter plugin required to be installed as part of IBM Security Guardium, Guardium Universal connector configuration. This gem is not a stand-alone program."
+pluginInfo.authors         = ['IBM', 'Rasika Shete']
+pluginInfo.email           = ['rasika.shete@ibm.com']
```
Comment (devanshah2) [inline filter-plugin/logstash-filter-dynamodb-guardium/build.gradle:16]: we can't have ibm emails in the files

File: filter-plugin/logstash-filter-dynamodb-guardium/build.gradle [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+// ===========================================================================
+// plugin info
+// ===========================================================================
+group                      'com.ibm.guardium.dynamodb' // must match the package of the main plugin class
+version                    "${file("VERSION").text.trim()}" // read from required VERSION file
+description                = "DynamoDB-Guardium filter plugin"
+pluginInfo.licenses        = ['Apache-2.0'] // list of SPDX license IDs
+pluginInfo.longDescription = "This gem is a Logstash Dynamodb filter plugin required to be installed as part of IBM Security Guardium, Guardium Universal connector configuration. This gem is not a stand-alone program."
+pluginInfo.authors         = ['IBM', 'Rasika Shete']
```
Comment (devanshah2) [inline filter-plugin/logstash-filter-dynamodb-guardium/build.gradle:15]: should leave it as IBM only.

**Comment by devanshah2** [comment]
Also missing the Sign off as required by https://github.com/IBM/universal-connectors/blob/main/CONTRIBUTING.md#legal

File: filter-plugin/logstash-filter-dynamodb-guardium/build.gradle (hunk 1 above)
Comment (Tal-Daniel) [reply filter-plugin/logstash-filter-dynamodb-guardium/build.gradle]: Removed by Rasika.

**Review by ofer-haim — APPROVED** [review]

File: filter-plugin/logstash-filter-dynamodb-guardium/build.gradle (hunk 2 above)
Comment (Tal-Daniel) [reply filter-plugin/logstash-filter-dynamodb-guardium/build.gradle]: Done.

**Review by devanshah2 — APPROVED** [review]

**Pull request merged**